{{cookiecutter.__src_folder_name}}/.devcontainer/devcontainer.json

```diff
@@ -27,26 +27,9 @@
                 {% if cookiecutter.project_host == "aca" %}
                 "ms-azuretools.vscode-docker",
                 {% endif %}
-                {% if 'postgres' in cookiecutter.db_resource %}
-                "mtxr.sqltools",
-                "mtxr.sqltools-driver-pg",
-                {% endif %}
                 "bierner.github-markdown-preview"
             ],
             "settings": {
-                {% if 'postgres' in cookiecutter.db_resource %}
```

Comment:
What do you think of an issue to try to bring back sqltools in the future? It's really nice when it works, and I always show it at workshops, so it'd be handy if it was already installed. I know you saw some bugs around it though, so I understand removing it for now.